msg/templates/uorb_microcdr/microRTPS_client.cpp.em

```diff
@@ -89,11 +110,7 @@ void* send(void* /*unused*/)
     int loop = 0, read = 0;
     uint32_t length = 0;
     size_t header_length = 0;
-
-    /* subscribe to topics */
-@[for idx, topic in enumerate(send_topics)]@
-    uORB::Subscription @(topic)_sub{ORB_ID(@(topic))};
-@[end for]@
+    struct SendTopicsSubs subs;
```

Comment:
You need to allocate this dynamically, otherwise you continue to have the same problem.

Reply:
Interesting enough without dynamically allocating it, it works without increasing the stack. But I'll do that.